src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -93,7 +92,7 @@
                     <ComboBox x:Name="ColorSchemeComboBox"
                               ItemsSource="{x:Bind ViewModel.AllColorSchemes, Mode=OneWay}"
                               SelectedIndex="0"
-                              SelectionChanged="ColorSchemeSelectionChanged"
+                              SelectionChanged="{x:Bind ViewModel.ColorSchemeSelectionChanged}"

```

Comment:
In the future, we should be able to bind `SelectedItem` directly to a CSVM, right?

Reply:
Yep, changed